rotkehlchen/tests/unit/decoders/test_compound.py

```diff
@@ -16,7 +16,7 @@
 ADDY3 = '0xb99CC7e10Fe0Acc68C50C7829F473d81e23249cc'
 
 
-@pytest.mark.vcr()
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
```

Comment:
I am not sure if the apikey is the source of problem here. Are you sure about it?

Reply:
Initially this change wasn't part of the PR. But after running the CI, the this particular test failed... ran it locally and it also failed.

Then checked the same test in develop and it was having this `filter_query_parameters=['apikey']`

### Develop
https://github.com/prettyirrelevant/rotki/blob/eb8de3d4f0b93251eda8ea7ec4a0ce73fcc77864/rotkehlchen/tests/unit/decoders/test_compound.py#L29

### Bugfixes
https://github.com/prettyirrelevant/rotki/blob/a61f978d3a2e2780c8bd3b88e59f5a92cc259260/rotkehlchen/tests/unit/decoders/test_compound.py#L19

Reply:
Ouff alright then. How did this discrepancy appear thought? :thinking:

@yabirgb pinging you since you seem to be the author of the compound tests